crates/client/rpc/src/tests.rs

```diff
@@ -0,0 +1,125 @@
+use mp_starknet::execution::types::{ContractAddressWrapper, Felt252Wrapper};
```

Comment:
the file hierary could be:
src/
   events/
     mod.rs
     tests.rs
     
     
Because all the tests in this file are event related

Reply:
makes sense! fixed.